index.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
-const {URL} = require('url');
-const urlRegex = require('url-regex');
+const urlRegex = require('url-regex-safe');
 const normalizeUrl = require('normalize-url');
 
 const getUrlsFromQueryParams = url => {
 	const ret = new Set();
-	const {searchParams} = (new URL(url.replace(/^(\/\/|(www\.))/i, 'http://$2')));
+	const {searchParams} = (new URL(url.replace(/^(?<slashes>\/\/|(?<identifier>www\.))/i, 'http://$2')));
```

Comment:
Don't make them named groups if you're not actually using the values.

Reply:
It seemed useless to me too, but ava was being prickly about this and testing kept throwing errors until I named the groups.